modules/serverless-domain-mapping-odc-developer.adoc

```diff
@@ -23,7 +23,7 @@ Your custom domain must point to the IP address of the {ocp-product-title} clust
 
 . Navigate to the *Topology* page.
 
-. Right-click on the service that you want to map to a domain, and select the *Edit* option that contains the service name. For example, if the service is named `example-service`, select the *Edit example-service* option.
+. Right-click on the service that you want to map to a domain, and select the *Edit* option that contains the service name. For example, if the service is named `showcase`, select the *Edit showcase* option.
```

Comment:
Delete "on," leaving "Right clickthe service you want...."

Reply:
Done.